draft-sharabayko-mops-srt.md

```diff
@@ -3003,6 +2999,63 @@ IV = (MSB(112, Salt) << 2) XOR (PktSeqNo)
 
 PktSeqNo is the value of the Packet Sequence Number field of the SRT data packet.
 
+# Best Practices and Configuration Tips for Data Transmission via SRT {#best-practices}
+
+## Live Streaming Use Case {#live-streaming-use-case}
+
+This section describes real world examples of live audio/video streaming and the current consensus maintaining the compatibility between SRT implementations by different vendors. It is meant as a guidance for developers to write applications compatible with existing SRT implementations.
+
+The term "live streaming" refers to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HLS protocol {{RFC8216}} is not part of this use case.
+
+The default SRT data transmission mode for continuous live streaming is message mode ({{transmission-mode-msg}}) with certain settings applied and described below:
+
+- Only data packets with their PP field set to "11b" are allowed, meaning a single data packet forms exactly one message ({{data-pkt}}).
+
+- Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop
+  (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be enabled.
+
+- Live congestion control (LiveCC) ({{liveCC}}) must be used.
+
+- The Order Flag ({{data-pkt}}) needs special attention. In the case
+  of live streaming, it is set to 0 allowing out of order
+  delivery of a packet. However, in this use case the Order Flag has to be ignored
+  by the receiver. As TSBPD is enabled, the receiver will still deliver
+  packets in order, but based on the timestamps. In the case of a
+  packet arriving too late and skipped by TLPKTDROP mechanism, the
+  order of delivery is still maintained except for potential sequence
+  discontinuity.
+
+  This method has grown historically and is the current common standard
+  for live streaming across different SRT implementations. A change or
+  variation of the settings will break compatibility between two parties.
+
+This combination of settings allows live streaming with a constant latency ({{srt-latency}}). The receiving end will not "fall behind" in time by waiting for missing packets. However, the data integrity might not be ensured, if packets or retransmitted packets do not arrive within the expected time frame. Audio or video interruption can occur, but the overall latency is maintained and does not grow up over time whenever packets are missing.
+
+## File Transmission Use Case {#file-transmission-use-case}
+
+This section describes the use case of file transmission as well as provides configuration examples.
+
+The usage of both message and buffer modes ({{data-transmission-modes}}) is possible in this case. For both modes, Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be turned off while congestion control should be set to file transfer congestion control (FileCC) ({{fileCC}).
```

Comment:
```suggestion
The usage of both message and buffer modes ({{data-transmission-modes}}) is possible in this case. For both modes, Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be turned off, while congestion control must be set to "file transfer congestion control" (FileCC) ({{fileCC}).
```

Reply:
@stevomatthews Why do we put "file transfer congestion control" in quotation marks here? 

There are several places in the document where I refer to "file transfer congestion control". Could you please take a look whether quotation marks are required somewhere else?

The same question for "live congestion control" or LiveCC.

Reply:
I put the quotation marks here to indicate that file transfer congestion control is a specific mechanism (just T and TS are — these are written with initial caps, which makes it clear). On line 2243, FileCC is described:

Reply:
Yes, I've followed your suggestion and decided to use initial caps for both live congestion control and file transfer congestion control. I think we can spend some time and see whether there is a need for using initial capital letters for other terms, but during the next RFC iteration.